### ISSUE TITLE
Update schematics do not update the spartacus version in package.json

### ISSUE THREAD

**Issue by znikola** [SAP/spartacus#7316]
As reported [here](https://spartacus-storefront.slack.com/archives/CD16V16FR/p1587370051253500), when running `ng update @spartacus/schematics`  it does not install the latest `next.6` version (coming from `next.4`).
Furthermore it does not update any versions in the package.json.

**Comment by Xymmer** [comment]
if we have to, we can include this fix after first RC

**Comment by znikola** [comment]
Asked and waiting for some additional information [here](https://spartacus-storefront.slack.com/archives/CD16V16FR/p1587547858287000?thread_ts=1587370051.253500&cid=CD16V16FR) and [here](https://spartacus-storefront.slack.com/archives/CD16V16FR/p1587554431289200?thread_ts=1587370051.253500&cid=CD16V16FR)

**Comment by znikola** [comment]
Closing this issues, as no feedback was provided. We can look into it again, if we get some additional information.

I managed to reproduce this issue only in cases when:
- Customers want to install a version of Spartacus that's not the latest one - e.g. Spartacus `next.6` is the latest, but clients are installing `ng add @spartacus/schematics@next.4`. In this case, npm will actually install only `next.4` for `@spartacus/schematics`, but will install `next.6` (the latest) for all other `@spartacus` libraries. The reason for this is the carrot (`^`) symbol in the `package.json`. This is a bit misleading, as the versions in the `package.json` will be `next.4` for all `@spartacus` libs, but the libs that are actually installed will be the latest (again, except for the `@spartacus/schematics`).
- The clients are _not_ using `yarn.lock` or `package-lock.json` files - in this case, the carrot (`^`) sign in `package.json` will always install the latest minor version from npm.

When `ng update @spartacus/schematics --next` or `ng update @spartacus/schematics@next.6` is ran, what happens is that angular will only bump the version of `@spartacus/schematics`. All other libs versions will be untouched (as the latest is already installed in `node_modules`). The `package.json` will still have the older version, which is a bit miss-leading, but that's just how npm works for packages that are using `^` in front of their versions.
